gno.land/cmd/gnoland/testdata/assertorigincall.txtar

```diff
@@ -133,24 +133,25 @@ func B() {
 func C() {
 	std.AssertOriginCall()
 }
--- r/foo/foo.gno --
-package foo
+-- p/demo/bar/bar.gno --
```

Comment:
Sorry, I realize I'm a bit picky here; but could you move `p/demo/bar/bar.gno` below `r/foo/foo.gno`?

It's hard to figure out what's going here, because the diff mixes them up. Thanks!

Reply:
No worries, [3f61b30](https://github.com/gnolang/gno/pull/3042/commits/3f61b3035afd3116d278d60dc13bb74249279f5b)